src/vmecpp/cpp/vmecpp/vmec/output_quantities/output_quantities.h

```diff
@@ -1003,13 +1003,13 @@ struct WOutFileContents {
   int nfp;
   int mpol;
   int ntor;
-  bool lasym;
+  bool lasym{};
```

Comment:
if we have to initialize it let's give it a sensible default explicitly:


```suggestion
  bool lasym = false;
```